src/core_plugins/console/server/proxy_route.js

```diff
@@ -21,9 +21,19 @@ import Joi from 'joi';
 import Boom from 'boom';
 import Wreck from 'wreck';
 import { trimLeft, trimRight } from 'lodash';
+import { unicodeBytesToString } from './lib/decode';
 
 function resolveUri(base, path) {
-  return `${trimRight(base, '/')}/${trimLeft(path, '/')}`;
+  let pathToUse = `${trimRight(base, '/')}/${trimLeft(path, '/')}`;
+  const questionMarkIndex = pathToUse.indexOf('?');
```

Comment:
nit: You can use `String.prototype.includes` now, so no need for indexOf checks and extra variables.

Reply:
Doesn't work in IE11 though (we've had bugs related to that).

Reply:
That's odd... it should be in the babel polyfill we use. Anyway, something I can sort out separately from this PR.